lib/Braintree/Transaction.php

```diff
@@ -147,30 +148,30 @@
  * @property-read string $cvvResponseCode
  * @property-read string $id transaction id
  * @property-read string $amount transaction amount
- * @property-read Braintree\Transaction\AddressDetails $billingDetails transaction billing address
+ * @property-read \Braintree_Transaction_AddressDetails $billingDetails transaction billing address
```

Comment:
Most of the phpdoc in the other files use a namespaced variation for property types. Does this change help in a particular IDE or development environment for you?